### PR TITLE
[1862] Drag and drop not working in MyEvents panel

### PR DESCRIPTION
**Pull request by stanica**
IE9 doesn't support HTML5 drag and drop on divs. I wrapped the text that's supposed to be draggable with an anchor tag and set draggable to true.

### CODE REVIEW

**Comment by ghost** [comment]
#### From: Bot.io (butyr.org)

---

# Received

Command **cmd_check** from @stanica received. Current queue size: 1

Live output at: http://botio.butyr.org:8000/a5f86f29d80e05c/output.txt

**Comment by ghost** [comment]
#### From: Bot.io (butyr.org)

---

# Success

Full output at http://botio.butyr.org:8000/a5f86f29d80e05c/output.txt

Total script time: 1.30 mins

**Comment by ghost** [comment]
#### From: Bot.io (butyr.org)

---

# Received

Command **cmd_preview** from @stanica received. Current queue size: 0

Live output at: http://botio.butyr.org:8000/2d3c5a0e5c1a80d/output.txt

**Comment by ghost** [comment]
#### From: Bot.io (butyr.org)

---

# Success

Full output at http://botio.butyr.org:8000/2d3c5a0e5c1a80d/output.txt

Total script time: 1.27 mins

Server is running on http://pr690.butyr.org/templates/test.html

File: css/tray.less [hunk 1]
```diff
@@ -138,7 +138,8 @@
         color: #FFF;
       }
     }
-    .list-item {
+    .list-item, a {
+      text-decoration: inherit;
```
Comment (k88hudson) [inline css/tray.less]: why not just none

**Pull request merged**